curio/__init__.py

```diff
@@ -11,6 +11,7 @@
 from .workers import *
 from .network import *
 from .file import *
+from .tls import *
```

Comment:
+1 for this. I was also a bit surprised by this. This should not be a issue for users, as they wan't interact with this module directly, but this will definitely confuse someone who is willing to explore the internals of curio.